bemani/data/triggers.py

```diff
@@ -33,7 +33,7 @@ def broadcast_score(self, data: Dict[BroadcastConstants, str], game: GameConstan
             self.broadcast_score_discord(data, game, song)
 
     def broadcast_score_discord(self, data: Dict[BroadcastConstants, str], game: GameConstants, song: Song) -> None:
-        if game == GameConstants.IIDX:
+        if game in [GameConstants.IIDX, GameConstants.POPN_MUSIC] :
```

Comment:
Not super big deal but suggest set notation instead of list notation since membership is `O(1)` in sets.